### PR TITLE
[cling] remove remnant declaration of removed function

### PR DESCRIPTION
**Pull request by ferdymercury**
# This Pull request:

## Changes or fixes:

Function was removed here https://github.com/root-project/cling/commit/c23abbc88b6ee61b966291c785794743b1bd5c62

Fixes https://github.com/root-project/cling/issues/456

## Checklist:

- [ ] tested changes locally
- [ ] updated the docs (if necessary)

### CODE REVIEW

**Review by hahnjo — APPROVED** [review]
LGTM

**Review by vgvassilev — APPROVED** [review]

**Review by vgvassilev — COMMENTED** [review]
The change does not correspond to the commit log.

File: interpreter/cling/docs/ReleaseNotes.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ describe the status of Cling in some detail, including major
 improvements from the previous release and new feature work.
 
 Note that if you are reading this file from a git checkout or the main
-[Cling web page](https://rawgit.com/root-project/cling/master/www/index.html),
+[Cling web page](https://root.cern/cling/),
```
Comment (vgvassilev) [inline interpreter/cling/docs/ReleaseNotes.md:11]: My comment was for this line.

Comment (ferdymercury) [reply interpreter/cling/docs/ReleaseNotes.md:11]: Oh. Should I change to the Github repo instead? Rawgit is discontinued and will shtit down at some point. Or how should I amend the commit message?

Comment (vgvassilev) [reply interpreter/cling/docs/ReleaseNotes.md:11]: We can host the static webpage of cling via github nowadays. Probably that's the best way to go here.

Comment (ferdymercury) [reply interpreter/cling/docs/ReleaseNotes.md:11]: So shall I point instead to https://github.com/root-project/cling/tree/master/www or to a future root-project.github.io/cling ?

Comment (ferdymercury) [reply interpreter/cling/docs/ReleaseNotes.md]: I've split the PR to facilitate.

**Review by vgvassilev — APPROVED** [review]
LGTM!

**Pull request merged**